### PR TITLE
switch to pnpm workspaces

### PR DESCRIPTION
**Pull request by dayhaysoos**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@web5/api": "0.8.1"
```
Comment (leordev) [inline package.json:3]: Why is it in the root level?

File: webpack/dist/index.html [hunk 2]
```diff
@@ -1,13 +0,0 @@
-<!DOCTYPE html>
```
Comment (leordev) [inline webpack/dist/index.html:1]: Q: Why is this file removed?

File: package.json (hunk 1 above)
Comment (dayhaysoos) [reply package.json:3]: For shared dependencies across all versions with pnpm workspace. We don't need to manage dependencies across all packages, we can just update this one source.

File: webpack/dist/index.html (hunk 2 above)
Comment (dayhaysoos) [reply webpack/dist/index.html:1]: It's a build file, it's still there but I think it got "removed" from git ignore

**Review by leordev — APPROVED** [review]
🎉🚀
